src/components/loader-bar/loader-bar-test.stories.tsx

```diff
@@ -30,3 +33,16 @@ DefaultStory.storyName = "default";
 export const LoaderBarComponentTest = (props: LoaderBarProps) => {
   return <LoaderBar mt={2} size="medium" {...props} />;
 };
+
+export const HeightFix: ComponentStory<typeof LoaderBar> = () => {
```

Comment:
**suggestion/nit-pick:** Might be better to call this example something like `LoaderBarWithMinHeight` or `WithMinHeight`. It just makes it easier to understand what the example is without having to dig too much in to the code of the example.